node/src/components/consensus/era_supervisor.rs

```diff
@@ -685,14 +719,18 @@ where
                             .cloned()
                             .expect("switch block should have era validator weights"),
                         switch_block.height() + 1,
-                        *switch_block.state_root_hash(),
+                        *switch_block.state_root_hash(), /* TODO: wrong if we want to cater to
+                                                          * upgrades modifying global state */
```

Comment:
Please create a ticket!

Reply:
I think that should be covered by HWY-248, actually.

Reply:
Is the ticket created? Can I resolve the comment?